spec/Procedures.tex

```diff
@@ -585,11 +607,44 @@ \subsubsection{The Default Intent}
 \chpl{ref} and \chpl{const ref} is similar to and interacts with return
 intent overloads (see \rsec{Return_Intent_Overloads}).
 
+\subsubsection{Default Intent for \chpl{owned} and \chpl{shared}}
+\label{Default_Intent_for_owned_and_shared}
+\index{intents!default for owned}
+\index{intents!default for shared}
+
+The default intent for \chpl{owned} and \chpl{shared} arguments is
+generally \chpl{in}.
+
```

Comment:
Should we indicate that the paragraph below is "an exception" to the paragraph above?